README.md

```diff
@@ -1,519 +1,136 @@
+<p align="center">
+  <br/>
+    <img alt="lighteval library logo" src="./assets/lighteval-doc.svg" width="376" height="59" style="max-width: 100%;">
+  <br/>
+</p>
+
+
+<p align="center">
+    <i>Your go-to toolkit for lightning-fast, flexible LLM evaluation, from Hugging Face's Leaderboard and Evals Team.</i>
```

Comment:
Could replace Hugging Face by the emoji/add the emoji somewhere

Reply:
I don't think it looks very good, moreover I like to have HuggingFace written here for SEO reasons.

Reply:
![Screenshot 2024-09-23 at 15 05 01](https://github.com/user-attachments/assets/5d8fba6c-061f-4844-8da1-97336fc08513)

Reply:
Agree!